packages/shared/src/storageChecker/storageChecker.ts

```
@@ -0,0 +1,74 @@
+import { debounce } from 'lodash';
+
+import appGlobals from '../appGlobals';
+import { SystemDiskFullError } from '../errors';
+import { EAppEventBusNames } from '../eventBus/appEventBusNames';
+import platformEnv from '../platformEnv';
+
+const diskFullErrorMessage = `Failed to execute 'transaction' on 'IDBDatabase': The database connection is closing`;
+
+function handleDiskFullError(error: unknown) {
+  const err = error as Error | undefined;
+  if (err && err?.message && err?.message.includes(diskFullErrorMessage)) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)